js/google_ima.js

```diff
@@ -52,6 +52,7 @@ require("../html5-common/js/utils/utils.js");
       //Constants
       var DEFAULT_IMA_IFRAME_Z_INDEX = 10004;
       var DEFAULT_ADS_REQUEST_TIME_OUT = 15000;
+      var DEFAULT_LOAD_VIDEO_TIME_OUT = 15000;
```

Comment:
There are a lot of timeouts in play and its hard to sync them all up. The 15 second ad request timeout is something we handle in the IMA plugin. The IMA SDK has a vastLoadTimeout for each wrapper:

https://developers.google.com/interactive-media-ads/docs/sdks/html5/v3/apis#ima.AdsRequest.vastLoadTimeout

and theres the loadVideoTimeout. We can revisit syncing all of these later.